src/components/form-editor/form-editor.component.tsx

```diff
@@ -477,4 +431,16 @@ function BackButton() {
   );
 }
 
+function FormEditor() {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Header title={t('schemaEditor', 'Schema editor')} />
+      <BackButton t={t} />
```

Comment:
I've lifted these components up so that changes to the FormEditor content don't trigger re-rendering the page header and other content above the fold.